Gemfile

```diff
@@ -5,7 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-ruby '>= 2.5', '< 2.7'
+ruby '2.5.1'
```

Comment:
I think @schneems added this so that we can experiment with Ruby trunk or Ruby 2.6 previews without changing Gemfile all the time.